app/views/pages/localhost.html.erb

```diff
@@ -7,13 +7,13 @@
   <p>
     That's totally fine, but this site uses the referrer to insert you into the webring.
     That means that clicking your hotline webring links would add
-    <code>http://localhost</code> as your entry in the webring, which is
-    definitely not what you want.
+    something like <code>http://localhost</code> as your entry in the webring,
+    which is definitely not what you want.
   </p>
   <p>
-    To keep localhost out of the webring, we ignore requests from localhost and
-    don't add them to the webring.  But don't worry! When you click on your real
-    site it'll work!
+    To keep local hosts out of the webring, we ignore requests from localhost
+    (and a few others, like <code>127.0.0.1</code>) and don't add them to the
```

Comment:
I actually intentionally left this vague -- we'll probably always block localhost, and this lets people know that we also block a few other similar hosts while letting us get away with not enumerating all of them.